packages/app/package.json

```diff
@@ -31,16 +31,15 @@
     "ci:development": "CYPRESS_BASE_URL=https://viron.work npm run test:e2e",
     "ci:development:record": "CYPRESS_BASE_URL=https://viron.work npm run test:e2e:record",
     "ci:production": "CYPRESS_BASE_URL=https://viron.app npm run test:e2e",
-    "ci:production:record": "CYPRESS_BASE_URL=https://viron.app npm run test:e2e:record",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "ci:production:record": "CYPRESS_BASE_URL=https://viron.app npm run test:e2e:record"
   },
   "lint-staged": {
     "**/*.{js,jsx,ts,tsx}": [
       "prettier --write",
       "eslint"
     ]
   },
+  "//": "style-to-object is an unnecessary dependency due to a Gatsby-Docusaurus version mismatch; we plan to remove it.",
```

Comment:
It appears to be an unfortunate bug that arises when running Gatsby and Docusaurus within the same monorepo. 

Previously, style-to-object happened to work due to de-duplication and being placed at the root, but with the recent changes, dependency resolution is now failing.